src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+    private static final String BYE_USAGE = "bye: to end the program\n";
+    private static final String LIST_USAGE = "list: to view the list of tasks\n";
+    private static final String MARK_USAGE = "mark: to mark a task as done";
+    private static final String MARK_FORMAT = "Format: mark {Number}\n";
+    private static final String UNMARK_USAGE = "unmark: to unmark a task as not done yet";
+    private static final String UNMARK_FORMAT = "Format: unmark {Number}\n";
+    private static final String TODO_USAGE = "todo: to add a todo task in your list";
+    private static final String TODO_FORMAT = "Format: todo {your task}\n";
+    private static final String DEADLINE_USAGE = "deadline: to add a deadline task in your list";
+    private static final String DEADLINE_FORMAT = "Format: deadline {your task} /by {deadline date}\n";
+    private static final String EVENT_USAGE = "event: to add an event task in your list";
+    private static final String EVENT_FORMAT = "Format: event {your task} /from {begin date} /to {end date}\n";
+    private static final String DELETE_USAGE = "delete: to delete a task";
+    private static final String DELETE_FORMAT = "Format: delete {Number}\n";
+    private static final String ADDING_TASK = "Got it. I've added this task:";
+    private static final String BIG_NUMBER = "The task number is bigger than the number of tasks";
+    private static final String FINISH_UNMARKING_MESSAGE = "Ok! I've marked this task as not done yet:";
+    private static final String FINISH_MARKING_MESSAGE = "Nice! I've marked this task as done:";
+    private static final String ASKING_MESSAGE = "What can I do for you?";
+    public static final String INCORRECT_KEYWORD = "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    public static final String EMPTY_TODO_DESCRIPTION = "OOPS!!! The description of a todo cannot be empty.";
+    public static final String EMPTY_DEADLINE_DESCRIPTION = "OOPS!!! The description of a deadline cannot be empty.";
+    public static final String EMPTY_EVENT_DESCRIPTION = "OOPS!!! The description of a event cannot be empty.";
+    public static final String INVALID_INPUT = "This is an invalid input, please follow this input format\n";
+    public static final String DEADLINE_INVALID_INPUT = INVALID_INPUT + DEADLINE_FORMAT;
+    public static final String EVENT_INVALID_INPUT = INVALID_INPUT + EVENT_FORMAT;
+    public static final String MEANINGLESS_SENTENCE_AFTER_KEYWORD = "OPPS!!! The sentence after keyword has no meaning";
+    public static final String REMOVE_MESSAGE = "Noted. I've removed this task:";
+    public static final String WRITEFILE_EXCEPTION_MESSAGE = "OPPS!!! Something went wrong when you write to data file";
+    public static final ArrayList<String> KEYWORDS = new ArrayList<>(
+            List.of("todo","deadline","event","list","bye","mark","unmark","help","delete")
+    );
+
+    public static void main(String[] args) {
```

Comment:
The entire main method is very long. Consider refactoring them into multiple methods to make it more readable. You may also try to apply the SLAP concept to make your main method looks more readable.